pkg/k8s/apis/cilium.io/v1alpha1/types.go

```diff
@@ -55,7 +55,7 @@ type KProbeArg struct {
 	// +kubebuilder:validation:Minimum=0
 	// Position of the argument.
 	Index uint32 `json:"index"`
-	// +kubebuilder:validation:Enum=auto;int;uint32;int32;uint64;int64;char_buf;char_iovec;size_t;skb;sock;string;fd;file;filename;path;nop;bpf_attr;perf_event;bpf_map;user_namespace;capability;kiocb;iov_iter;cred;load_info;module;syscall64;
+	// +kubebuilder:validation:Enum=auto;int;uint32;int32;uint64;int64;char_buf;char_iovec;size_t;skb;sock;string;fd;file;filename;path;nop;bpf_attr;perf_event;bpf_map;user_namespace;capability;kiocb;iov_iter;cred;load_info;module;syscall64;linux_binprm
```

Comment:
As you are changing the CRD you have also to update the version of that in https://github.com/cilium/tetragon/blob/64e5baf632be3ab62b2335f1f21c75efce5a762f/pkg/k8s/apis/cilium.io/v1alpha1/version.go#L10 (i.e. from 1.1.3 to 1.1.4).